test/src/module/command/backupTest.c

```diff
@@ -1428,6 +1428,7 @@ testRun(void)
 
         time_t timestamp = 1575401652;
         String *backupLabel = backupLabelFormat(backupTypeFull, NULL, timestamp);
+        const String *const olderBackupLabel = backupLabelFormat(backupTypeFull, NULL, timestamp - 3);
```

Comment:
Can you clarify why we've changed `- 2` to `- 3`?

Reply:
-2 is used in line 1485, -1 - in 1504. Diff and incr backups should be after full backup. So -3 is used for full backup